index.d.ts

```diff
@@ -1,3 +1,24 @@
+interface Options {
+	/**
+	When true, Leven will return the value of `maxDistance` if the calculated Levenshtein distance is greater than `maxDistance`.
```

Comment:
I think this should more clearly describe the purpose of the option. Should also include a use-case.